apps/src/lib/util/javascriptMode.js

```diff
@@ -65,11 +65,15 @@ export function apiValidateType(
         break;
       case 'pinid':
         var reservedPins = ['A2', 'A3', 'A7', 1, 9, 10];
```

Comment:
Can we align on the premise that reserved pins _are_ valid pins, even if we don't want students to use them? It seems like we'd first want to check for errors (invalid pins), and then check for warnings (reserved pins). We could flip the order of the two `if` statements below if we added each "reserved" value to `validPins`.
I feel like this might make the code easier to read and understand. For students, it would result in mostly the same experience.

Reply:
This definitely makes sense - I agree that reserved pin ids are valid as well so edited code to reflect that. Thanks!